about.html

```diff
@@ -12,53 +12,100 @@
 </head>
 
 <body>
-    <div class="container about-page">
-     
-    About
 
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
 
 
-    <img src="img/about-plan.png" alt="strategy">
+    <div class="container about-page">
+
+      <section class="top-content">
+
+        <header>
+          <nav>
+            <div class="logo">
+                <img src="img/lambda-black.png" alt="Lambda">
+            </div>
+            <div class="links">
+            <a href="../web-Sprint-Challenge-User-Interface-and-Git/index.html">Home</a>
+            <a href="../web-Sprint-Challenge-User-Interface-and-Git/about.html">About</a>
+            <a href="#">Products</a>
+            <a href="#">Blog</a>
+            <a href="#">Contact</a>
+        </div>
+        </nav>
     
```

Comment:
You can also use <section> for each of the 4 middle content boxes, strategy, how we work, places we work and collaboration.  It is more descriptive than divs.